### DISCUSSION TITLE
Welcome to tkmacosx Discussions!

### DISCUSSION THREAD

**Discussion by Saadmairaj** [Saadmairaj/tkmacosx#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TomOfHelatrobus** [comment]
Regarding the button widget, is it possible to align the text inside the widget to the right or left? All I see is that text automatically goes to the center.

**Reply by Saadmairaj** [reply]
Yes, it is possible to align the text by using the `anchor` option for the button you can set the text to any direction like *n, s, e, w, se, sw, ne, nw*. But currently, there is an issue while setting anchor attribute which can be fixed by installing the library from GitHub, refer to this [issue](https://github.com/Saadmairaj/tkmacosx/issues/7#issuecomment-733830018).

Example:
```python 
import tkinter as tk
import tkmacosx as tkm

root = tk.Tk()
b1 = tk.Button(root, text='Left', anchor='w')
b2 = tk.Button(root, text='Right', anchor='e')
b1.pack()
b2.pack()
root.mainloop()
```

**Reply by TomOfHelatrobus** [reply]
That did the trick. I noticed that the width is also no longer measured in pixels and is much closer to comparable widths in Windows. That's a plus. I don't see the background colors though. I'll be watching for any updates. Thanks for writing this. I'm writing a small program to help union stewards in the post office enforce scheduling provisions in the contract. I've gotten many request for a Mac version and this program looks like it can help. If you take suggestions, the selectcolor attribute of the radiobutton when indicatoron=0 doesn't work properly in Mac. You might be the man to fix it.

**Reply by Saadmairaj** [reply]
Please explain _**I don't see the background colors though**_? If there is an issue you can [submit it here](https://github.com/Saadmairaj/tkmacosx/issues/new/choose), so I can solve it. Also, thanks for the suggestion I'll look into it.
